lib/core-common/src/utils/to-require-context.ts

```diff
@@ -25,21 +25,33 @@ const detectBadGlob = (val: string) => {
 const isObject = (val: Record<string, any>) =>
   val != null && typeof val === 'object' && Array.isArray(val) === false;
 
+const dirname = (pattern: string) => {
+  if (pattern.slice(-1) === '/') return pattern;
```

Comment:
what if the user passes a directory but doesn't include a trailing slash? any chance you can provide a test case for this and for any other cases you considered in this logic?

Reply:
This was behaviour was something that I saw in the glob-base dependency that was behaving a bit strange. I have now checked with the implementation in storybook on how it currently behaves and I believe that this extra logic did not make any difference. I have removed this now.

I think the current test are already very good but I added two more that focus on the folder pattern.